SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -455,26 +443,16 @@ void USpatialWorkerConnection::OnConnectionSuccess()
 	}
 
 	OnConnectedCallback.ExecuteIfBound();
-	GameInstance->HandleOnConnected();
-}
-
-void USpatialWorkerConnection::OnPreConnectionFailure(const FString& Reason)
-{
-	bIsConnected = false;
-	GameInstance->HandleOnConnectionFailed(Reason);
 }
 
 void USpatialWorkerConnection::OnConnectionFailure()
 {
 	bIsConnected = false;
 
-	if (GEngine != nullptr && GameInstance->GetWorld() != nullptr)
-	{
-		uint8_t ConnectionStatusCode = Worker_Connection_GetConnectionStatusCode(WorkerConnection);
-		const FString ErrorMessage(UTF8_TO_TCHAR(Worker_Connection_GetConnectionStatusDetailString(WorkerConnection)));
+	uint8_t ConnectionStatusCode = Worker_Connection_GetConnectionStatusCode(WorkerConnection);
+	const FString ErrorMessage(UTF8_TO_TCHAR(Worker_Connection_GetConnectionStatusDetailString(WorkerConnection)));
```

Comment:
Should we check to make sure `WorkerConnection` is valid and not null before doing this?

Reply:
It's a nullptr to a c_worker struct, so I don't think we can check the validity, however, I'll add a nullptr check.

Reply:
Added nullptr check.